docs-user/memory-allocations.md

```diff
@@ -40,17 +42,23 @@ The allocations can be viewed in the call tree, and flame graph, but not the sta
 
 ![A screenshot of the UI to turn on native allocations.](images/allocation-feature.png)
 
-### Limitations in Native Allocation Tracking
+### Native allocation summary strategies
+
+#### Summarize Retained allocations
+
+In the profiler, the allocations are sampled, but de-allocations are matched against the sampled allocations. This means that the profiler can tell which allocations were retained, and which were not in a given time frame. In the UI, drag an interactive range selection to update which allocations were retained within the view. This can be helpful to identify potential memory leaks. For instance in [this profile of DevTools opening and closing](https://perfht.ml/2LKZsfY) you can create a range selection and scrub through the different parts of the profile. Ideally, after the window is closed, most of the allocations should be removed.
+
+#### Summarize Allocations
 
-#### Limitation: Allocations tracked per-thread
+This option in the dropdown shows all allocations that happened, regardless if they were deallocated.
```

Comment:
Reading this text, it's not obvious that they're still sampled in this mode. Maybe it should be made clearer in this paragraph too. (same for the deallocations below)

Reply:
I reworded it to include "sampled"